### ISSUE TITLE
Rendering text dynamically via {eval} returns single quoted output of strings.

### ISSUE THREAD

**Issue by mcauthorn** [executablebooks/jupyterlab-myst#99]
### Describe the bug

**Context**
When rendering strings using {eval}, it returns single quoted text. Integers render as expected (see below).

**Expectation**

The behavior should be more like a normal print, which strips the quotes, e.g.
```python
print("foo") # This is what we should see
print("foo".__repr__()) # This is what a rendered {eval} with a string looks like.
```
**Note**:  integers render as expected given, e.g.

```python
test1 = 12 
test2 = "Test String"
```
....Followed by the following MyST markdown:

```
- Testing an int:   {eval}`test1`
- Testing a string: {eval}`test2`
```
...Renders the following. Notice the quotes around the string variable. 

<img width="336" alt="image" src="https://user-images.githubusercontent.com/2439265/221356715-7bacb936-61cf-46e2-8d80-b889fdd6b6c2.png">

**Version Info.**

Jupyterlab version: 3.6.1
jupyterlab-myst:      1.1.0

### Reproduce the bug

To reproduce, use {eval} with strings vs. ints.

**Comment by rowanc1** [comment]
Thank you @mcauthorn -- @agoose77, I think that the previous implementation had a text renderer that stripped quotes. I was also talking to @fperez about this last night. It is relatively safe to remove the quotes in text, as you can put them in markdown if they are desirable.

From the implementation point of view, I think that we should inspect the mime-bundle, and if we are going to use a text output, then this should use native react to render, with the quote-stripping behaviour.

**Comment by agoose77** [comment]
> From the implementation point of view, I think that we should inspect the mime-bundle, and if we are going to use a text output, then this should use native react to render, with the quote-stripping behaviour.

Yes!

**Comment by rowanc1** [comment]
@mcauthorn @fperez, this has just been released by @agoose77 as v1.1.1 🚀🎉

**Comment by fperez** [comment]
Awesome, just updated our hub, big thanks!!
